src/main/java/com/blackducksoftware/integration/hub/service/HubMultipleViewTransformer.java

```diff
@@ -40,40 +40,40 @@
 
 import okhttp3.Response;
 
-public class HubResponseItemsManager {
-    private final HubResponseItemManager hubResponseItemManager;
+public class HubMultipleViewTransformer {
```

Comment:
For a class name change what about HubViewCollectionTransformer.  It will produce a collection of view objects.